src/cache/local.ts

```diff
@@ -61,17 +61,27 @@ export class LocalCache<T = unknown> implements Cache<T> {
     }
   }
 
-  async set(key: string, value: Readonly<T>, ttl: number): Promise<void> {
+  async set(key: string, value: T, ttl: number): Promise<void> {
```

Comment:
The `Readonly` here made sense, was it removed for a particular reason?

Reply:
It was removed since i couldn't type cast `value as AdapterResponse` with `Readonly`. The other option was to convert it first to `unknown`

Reply:
I'd rather we cast to unknown in that case. We know what we're doing in here, but whoever uses the cache should be under strict types if possible